### DISCUSSION TITLE
In React Native (Expo), when using atomWithQuery the data field is empty

### DISCUSSION THREAD

**Discussion by corasan** [pmndrs/jotai#1977] [Bug report]
I am using `useQuery` to fetch data, but I'm trying to switch to `atomWithQuery`. When using regular `useQuery` the data field is not empty, as expected, but when using `atomWithQuery` data is empty until I re-render the screen. It seems as if when the request succeeds the component doesn't re-render.

```typescript
const [dataAtom] = atomWithQuery((get) => ({
  queryKey: ['myKey'],
  queryFn: async () => {
    const params = get(paramsAtom)
    const res = await axios.get(url, {
      params,
    });
    return res.data
  }
}))
```

**Comment by dai-shi** [comment]
Can you create a minimal reproduction with dummy `queryFn` (no axios) with https://csb.jotai.org ?

**Comment by t6adev** [comment]
Closing this due to a lack of response from the owner for an extended period.
Please feel free to reopen if the issue persists.
